Localisation/NumberToWords/SpanishNumberToWordsConverter.ts

```diff
@@ -0,0 +1,106 @@
+module Humanizer.Localisation.NumberToWords {
+    "use strict";
+
+    interface ExceptionsDictionary {
+        [num: number]: string;
+    }
+
+    var unitsMap = [
+        "cero", "uno", "dos", "tres", "cuatro", "cinco", "seis", "siete", "ocho", "nueve",
+        "diez", "once", "doce", "trece", "catorce", "quince", "dieciséis", "diecisiete", "dieciocho", "diecinueve",
+        "veinte", "veintiuno", "veintidós", "veintitrés", "veinticuatro", "veinticinco", "veintiséis", "veintisiete", "veintiocho", "veintinueve"];
+    var tensMap = ["cero", "diez", "veinte", "treinta", "cuarenta", "cincuenta", "sesenta", "setenta", "ochenta", "noventa"];
+    var hundredsMap = ["cero", "ciento", "doscientos", "trescientos", "cuatrocientos", "quinientos", "seiscientos", "setecientos", "ochocientos", "novecientos"];
+
+    function getUnitValue(num: number, isOrdinal: boolean): string {
```

Comment:
I see that here is where the exception is thrown if Ordinal is requested, but this method is never called. So is ordinal supported now?